cli/src/lockfiles/python.rs

```diff
@@ -104,7 +104,9 @@ impl Parse for Poetry {
             .packages
             .drain(..)
             .filter(|package| {
-                package.source.as_ref().map_or(true, |source| source.source_type == "git")
+                package.source.as_ref().map_or(true, |source| {
+                    source.source_type == "git" || source.source_type == "legacy"
+                })
```

Comment:
We do actually support submitting `git` dependencies to the backend, right? It seems like that's incorrectly disabled right now?

We might want to open an issue for supporting git dependencies with Poetry, not sure if that's desired?